### PR TITLE
Mj add epic choice cards

### PR DESCRIPTION
**Pull request by michaelbjacobson**
Adds ability to enable choice cards on epic variants

### CODE REVIEW

File: app/models/EpicTests.scala [hunk 1]
```diff
@@ -69,7 +70,7 @@ case class EpicTest(
   sections: List[String] = Nil,
   excludedTagIds: List[String] = Nil,
   excludedSections: List[String] = Nil,
-  alwaysAsk: Boolean = false,
+  alwaysAsk: Boolean = false,loads
```
Comment (TomPretty) [inline app/models/EpicTests.scala:73]: I think this is maybe a typo?

Comment (michaelbjacobson) [reply app/models/EpicTests.scala]: ✅

**Review by tomrf1 — APPROVED** [review]
looks good!

**Review by TomPretty — APPROVED** [review]
Looks great! Nice work 👍

File: public/src/components/channelManagement/epicTests/epicTestChoiceCardsEditor.tsx [hunk 2]
```diff
@@ -3,26 +3,26 @@ import FormControlLabel from '@material-ui/core/FormControlLabel';
 import Checkbox from '@material-ui/core/Checkbox';
 
 interface EpicTestChoiceCardsEditorProps {
-  useChoiceCards?: boolean;
-  updateUseChoiceCards: (useChoiceCards?: boolean) => void;
+  showChoiceCards?: boolean;
+  updateshowChoiceCards: (showChoiceCards?: boolean) => void;
```
[the remark this answers is not in the document]
Comment (michaelbjacobson) [reply public/src/components/channelManagement/epicTests/epicTestChoiceCardsEditor.tsx:7]: Good spot, thanks @TomPretty! I changed `useChoiceCards` to `showChoiceCards` using the project-wide search and replace.

**Comment by michaelbjacobson** [comment]
[Build epic choice cards on AMP](https://trello.com/c/CsiIJ6WP/2800-build-epic-choice-cards-on-amp)

**Pull request merged**